src/addMeToYourFriends.ts

```diff
@@ -0,0 +1,146 @@
+import { html, TemplateResult } from "lit-html";
+import { styleMap } from "lit-html/directives/style-map";
+import { DataBrowserContext, LiveStore } from "pane-registry";
+import * as UI from "solid-ui";
+import { padding, textCenter } from "./baseStyles";
+
+let buttonContainer = <HTMLDivElement>document.createElement("div");
+//TODO create dedicated component in UI
+const positiveFrontendMessageDiv = <HTMLDivElement>document.createElement("div");
+positiveFrontendMessageDiv.setAttribute(
+  "style",
+  "margin: 0.1em; padding: 0.5em; border: 0.05em solid gray; background-color: #efe; color:black;"
+);
+
+const styles = {
+  button: styleMap({ ...textCenter(), ...padding() }),
+};
+
+//Same not logged in error message like on 'Chat with me' buton
+const userNotLoggedInErrorMessage = "Current user not found! Not logged in?";
+const internalErrorMessage = "An internal error occured!";

```

Comment:
I think I would pop the messages into another file as well.  this was originally a small pane, but as we are adding more functionality we should think about having folders.  So for instance you could create a friend folder under src and pop all the friend functionality in to there and then this message file would be part of that.

Reply:
I created a texts file for all messages but I did not add it to its own folder because I gathered also 2 messages from the Chat code. (so a text file for all texts in profile pane )